docs/topics/using/edgectl/service-preview-reference.md

```diff
@@ -669,6 +683,39 @@ spec:
 
 **Note**: If you already had an active Edge Control Daemon connection to the cluster, you must reconnect to the cluster for the Edge Control Daemon to detect the change to the Host resource. This limitation will be removed in the future.
 
+## Service Mesh Support
+
+Much like a service mesh, the Service Preview `traffic-agent` runs as a sidecar to the container running your application. Luckily, due to the magic of service mesh Pod networking, Service Preview will run alongside a service mesh without any changes needed to be made on your part.
```

Comment:
Delete "Much like a service mesh" because a service mesh isn't just the sidecar, it's also other stuff. So we can avoid the comparison.